package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "cast-and-crew",
+  "version": "1.0.0",
+  "description": "A module for displaying the works of a selected cast/crew member.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "server-dev": "nodemon server/index.js",
+    "react-dev": "webpack -d --watch",
+    "seed": "node -e 'require(\"./database/seeder.js\").seed()'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fec1-arwen/cast-and-crew.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fec1-arwen/cast-and-crew/issues"
+  },
+  "homepage": "https://github.com/fec1-arwen/cast-and-crew#readme",
+  "dependencies": {
+    "bluebird": "^3.5.5",
+    "express": "^4.17.1",
+    "mongoose": "^5.6.3",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.5.0",
```

Comment:
Babel and webpack don't seem to be `devDependencies` based on how you're using them; move them to `dependencies`.